deployment/docker-compose.yml

```diff
@@ -67,6 +67,30 @@ uwsgi:
   user: root
   restart: on-failure:5
 
+uwsgi-ephemeral:
+  # Note you cannot scale if you use conteiner_name
+  container_name: mezzanine-uwsgi-ephemeral
```

Comment:
Also maybe it is better to use generic names - so we can easily port logic like this to other projects. So ```uwsgi-ephemeral``` might be nicer....